persistence-cassandra/javadsl/src/multi-jvm/resources/logback.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding this log so that we can have more information about persistence-cassandra-* tests when they fail.